app/models/custom_actions/actions/custom_field.rb

```diff
@@ -28,7 +28,7 @@
 
 class CustomActions::Actions::CustomField < CustomActions::Actions::Base
   def self.key
-    :"custom_field_#{custom_field.id}"
+    custom_field.attribute_name.to_sym
```

Comment:
Yes, it could be done and would be correct from a method call perspective.
But from a semantic perspective, I'm not sure if `CustomActions::Actions::CustomField.key` should return the `CustomField#attribute_getter`. It probably won't hurt, but I'm not sure it reveals the intent of the key.
And BTW I did not find where and how this key is used exactly.

Same for the other occurences. 

Maybe a `#attribute_key` can be extracted and used wherever `attribute_name.to_sym` is used. It would better reveal the different usages of `"custom_field_#{id}` over the code base: sometimes as a string, sometimes as a symbol for key / getter / setter.

WDYT?

Reply:
I am fine with having `#attribute_key`, but I'm also fine with the way it is currently.